packages/realm-react/package.json

```diff
@@ -96,9 +96,9 @@
     "hooks"
   ],
   "author": {
-    "name": "Realm",
+    "name": "MongoDB",
```

Comment:
P.S. Some of the other package.jsons still have `"name": "Realm"`.

Reply:
Ah, I wasn't sure if I was sold on that yet, but I guess MongoDB is the official author, so I can update that as well